typescript/api-cors-lambda-crud-dynamodb/index.ts

```diff
@@ -14,62 +16,94 @@ export class ApiLambdaCrudDynamoDBStack extends cdk.Stack {
       },
       tableName: 'items',
 
-      // The default removal policy is RETAIN, which means that cdk destroy will not attempt to delete
-      // the new table, and it will remain in your account until manually deleted. By setting the policy to 
-      // DESTROY, cdk destroy will delete the table (even if it has data in it)
+      /**
+       *  The default removal policy is RETAIN, which means that cdk destroy will not attempt to delete
+       * the new table, and it will remain in your account until manually deleted. By setting the policy to
+       * DESTROY, cdk destroy will delete the table (even if it has data in it)
+       */
       removalPolicy: cdk.RemovalPolicy.DESTROY, // NOT recommended for production code
     });
 
-    const getOneLambda = new lambda.Function(this, 'getOneItemFunction', {
-      code: new lambda.AssetCode('src'),
-      handler: 'get-one.handler',
-      runtime: lambda.Runtime.NODEJS_10_X,
+    const getOneLambda = new NodejsFunction(this, 'getOneItemFunction', {
```

Comment:
If you are going to use direct component imports (ie. `import { NodejsFunction } from '...';` vs `import * as lambdanode from '...';` and `new NodejsFunction()` vs `new lambdanode.NodejsFunction()`) please do so for all other components (`lambda.Runtime`, `dynamodb.Table`, etc.)

Reply:
Just to clarify, I think importing components directly is a good approach for examples as it looks cleaner and is considered good practice for projects with few imports.